src/entities/Article.ts

```diff
@@ -0,0 +1,34 @@
+import { Field, ID, ObjectType } from 'type-graphql';
+import { prop as Property, getModelForClass } from '@typegoose/typegoose';
+
+@ObjectType({ description: 'The Article Model' })
+export class Article {
+  @Field(() => ID)
+  id: string;
```

Comment:
there's been a push to name the id field `uuid` and not `id`

Reply:
Fixed